src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -13,7 +13,12 @@ def call
     private
 
     def create_workflows
-      parsed_workflows_yaml = YAML.safe_load(File.read(@yaml_file))
+      begin
+        parsed_workflows_yaml = YAML.safe_load(File.read(@yaml_file))
+      rescue Psych::SyntaxError => e
+        raise WorkflowsYamlNotParsable, "Unable to parse .obs/workflows.yml: #{e.message}"
```

Comment:
This line is wrong:

```
NameError: uninitialized constant Workflows::YAMLToWorkflowsService::WorkflowsYamlNotParsable
```

We should have a spec to cover that this line correctly raises.